app/views/home/index.html.slim

```diff
@@ -1,2 +1,37 @@
-h1 Home#index
-p Find me in app/views/home/index.html.slim
+.container
+  h1 Investment tracking
+
+.entries
+  .container-fluid
+    .row
+      .col-md-2
+      .col-md-6
+        p Your open investment time in hours:
+      .col-md-2
+        p Your rate to total worked hours:
+      .col-md-2
+        p Your used investment time:
```

Comment:
These strings should be extracted into the translations file: http://guides.rubyonrails.org/i18n.html